lighthouse-core/gather/gatherers/full-page-screenshot.js

```diff
@@ -89,9 +91,27 @@ class FullPageScreenshot extends FRGatherer {
       screenOrientation: {angle: 0, type: 'portraitPrimary'},
     });
 
-    // TODO: elements collected earlier in gathering are likely to have been shifted by now.
-    // The lower in the page, the more likely (footer elements especially).
-    // https://github.com/GoogleChrome/lighthouse/issues/11118
+    // Now that the viewport is taller, give the page some time to fetch new resources that
+    // are now in view.
+    const networkMonitor = new NetworkMonitor(context.driver.defaultSession);
+    const waitForNetworkIdleResult = waitForNetworkIdle(session, networkMonitor, {
+      networkQuietThresholdMs: 1000,
+      busyEvent: 'network-2-busy',
+      idleEvent: 'network-2-idle',
+      isIdle: recorder => recorder.is2Idle(),
+    });
+    await Promise.race([
+      new Promise(resolve => setTimeout(resolve, 1000 * 10)),
```

Comment:
thoughts on how long timeout should be?

Reply:
how long did the networkidle promise take on CNN?  maybe double that?

Reply:
IMO it shouldn't be that long. FPS isn't critical information and I don't want to wait an extra 10s just so everything lines up. Quickly eyeballing it I would say 2s.

Reply:
cnn.com took ~1-1.5s to reach isCritical network quiet time on my machine

Settled on 5s because 1-2s is way too short to expect a network response in the extreme case

Reply:
5s seems reasonable to me! 

Quick FYI, some pages have carousels that animate so the full screen screenshot _may_ end up out of sync with the last screenshot from the timeline screenshots. I don't think that's a big deal, but it may lead to confusion of eagle-eyed report viewers :-)